shopinvader_v1_product/tests/test_backend.py

```diff
@@ -11,12 +11,6 @@ class BackendCase(CommonCase):
     @classmethod
     def setUpClass(cls):
         super(BackendCase, cls).setUpClass()
```

Comment:
A lot of file here import the file common, but the file is missing, maybe you forget to commit it